src/app/modules/shared_models/contacts_item.nim

```diff
@@ -8,19 +8,23 @@ type
   Item* = ref object
     pubKey: string
     name: string
+    localNickname: string
     icon: string
     isBlocked: bool
+    onlineStatus: bool
     isMutualContact: bool
     verificationState: ContactVerificationState
 
-proc initItem*(pubKey, name, icon: string, isMutualContact, isBlocked: bool, 
+proc initItem*(pubKey, name, localNickname, icon: string, isMutualContact, isBlocked, onlineStatus: bool, 
   isContactVerified, isContactUntrustworthy: bool): Item =
   result = Item()
   result.pubKey = pubKey
   result.name = name
+  result.localNickname = localNickname
   result.icon = icon
   result.isMutualContact = isMutualContact
   result.isBlocked = isBlocked
+  result.onlineStatus = onlineStatus
```

Comment:
@MishkaRogachev if you're setting online status here (from a constructor), and as I see all usages of `initItem` have online status set to false, and since I don't see any proc which is used for set online status I am wondering how do you change online status during the app run time when update about any of contacts' online status come to the app?

Reply:
@saledjenic here im just added online data field without behavior.  P.s with PR in not actual since https://github.com/status-im/status-desktop/issues/5951